stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -81,6 +81,7 @@ spec:
           ARGS=("--port" "${REDIS_PORT}")
           {{- if .Values.usePassword }}
           ARGS+=("--requirepass" "${REDIS_PASSWORD}")
+          ARGS+=("--masterauth" "${REDIS_PASSWORD}")
```

Comment:
I understand this address the issue in the case the original master is recreated as a slave.
However, when it's is started as a master node, does it affect the original behaviour? Do you know if there's any drawback on doing this?

Reply:
Hi @juan131 ,
I tested it and the behavior do not change when starting as master. The first time it is initialized as master without problems, after kill it, a new master is elected and the old one starts as slave. The workflow is correct and the old master can start as slave. Without the option the old master cannot start as slave because it hasn't the password so the pod fails.

Reply:
👍 Thanks for the detailed explanation!